src/splat.F

```diff
@@ -18,34 +18,29 @@
 C> if an equator point exists, its weight should be halved.
 C> This version invokes the ibm essl matrix solver.
 C>
-C> PROGRAM HISTORY LOG:
-C> -  96-02-20  IREDELL
-C> -  97-10-20  IREDELL  ADJUST PRECISION
-C> -  98-06-11  IREDELL  GENERALIZE PRECISION USING FORTRAN 90 INTRINSIC
-C> - 1998-12-03  IREDELL  GENERALIZE PRECISION FURTHER
-C> - 1998-12-03  IREDELL  USES AIX ESSL BLAS CALLS
-C> - 2009-12-27  DSTARK   updated to switch between ESSL calls on an AIX
+C> ### Program History Log
+C> Date | Programmer | Comments
+C> -----|------------|---------
+C> 96-02-20 | IREDELL |
```

Comment:
You need to fill all three fields of the table. So this will not work:

`C> 96-02-20 | IREDELL |`

Change it to:

`C> 96-02-20 | IREDELL | Initial.`

Now there is a third field.